src/reducer/WebSocketUtilityDispatchers.ts

```diff
@@ -0,0 +1,33 @@
+import { IMessage } from "react-stomp-hooks";
+import { Action } from "redux";
+import { ThunkDispatch } from "../util/Types";
+import { LongRunningTask } from "../model/LongRunningTask";
+import {
+  asyncActionSuccess,
+  asyncActionSuccessWithPayload,
+} from "../action/SyncActions";
+
+export function updateLongRunningTasks(message: IMessage, action: Action) {
+  return async (dispatch: ThunkDispatch) => {
+    if (message.headers["reset"]) {
+      dispatch(asyncActionSuccess(action));
+      return;
+    }
+
+    const tasks: { [uuid: string]: LongRunningTask } = JSON.parse(message.body);
+
+    const mapped = Object.keys(tasks).map((uuid) => {
+      const task = tasks[uuid];
+      if (task.startedAt) {
+        // @ts-ignore
+        task.startedAt = new Date(task.startedAt * 1000);
+        task.startedAt.setMilliseconds(0); // round to second
```

Comment:
If the `startedAt` value comes from the server, I propose to do the rounding on server. Java Time API has much nicer API for this and it would be more consistent from the clients' point of view.

Reply:
And if it doesn't come from the server, the rounding should be done where the `startedAt` value is initialized.